app/scripts/templates/sign_in_totp_code.mustache

```diff
@@ -0,0 +1,37 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="fxa-totp-code-header">{{#t}}Enter security code{{/t}}
+        {{#serviceName}}
+            <span class="service">{{#t}}Continue to %(serviceName)s{{/t}}</span>
+        {{/serviceName}}
+    </h1>
+  </header>
+
+  <section>
+    <div class="error"></div>
+    <div class="success"></div>
+
+    <div class="graphic graphic-two-factor-auth"></div>
+
+    <p class="verification-totp-message">
+    {{#t}}Open your authentication app and enter the security code it provides.{{/t}}
+    </p>
+
+    <form novalidate>
+      <div class="input-row token-code-row">
+        <input type="text" class="tooltip-below totp-code" placeholder="{{#t}}Enter 6-digit code{{/t}}" required autofocus />
```

Comment:
![image](https://user-images.githubusercontent.com/128755/37054664-758c8ad4-214d-11e8-9374-c5240a6a7265.png)

do you want to limit this to 6 digits or so? also limit this to numbers?